src/main/java/com/example/Tictactoe.java

```diff
@@ -0,0 +1,102 @@
+package com.example;
+
+public class Tictactoe {
+
+	public static char getWinner(char[][] board) {
+		validateBoard(board);
+
+		int checkRow1 = 0;
+		int checkRow2 = 0;
+		int checkRow3 = 0;
+		int checkColumn1 = 0;
+		int checkColumn2 = 0;
+		int checkColumn3 = 0;
+		int checkDiagonal1 = 0;
+		int checkDiagonal2 = 0;
+		int rowIndex;
+		int columnIndex;
+
+		//checking rows
+		rowIndex = 0;
+		while (rowIndex < 3) {
```

Comment:
The two `while` cycle contains very similar logic. Can we avoid somehow the duplication? Or merge the two cycle?